en/identity-server/7.0.0/mkdocs.yml

```diff
@@ -22,6 +22,12 @@ site_url: https://is.docs.wso2.com/en/7.0.0
 extra:
   base_path: /en/7.0.0
   is_version: 7.0.0
+  footer:
```

Comment:
Let's also add stackoverflow link for Asgardeo.

https://stackoverflow.com/questions/tagged/wso2-asgardeo

https://github.com/himeshsiriwardana/docs-is/blob/fixes-07-15/en/asgardeo/mkdocs.yml